packages/trpc/server/admin-router/router.ts

```diff
@@ -0,0 +1,33 @@
+import { TRPCError } from '@trpc/server';
+
+import { isAdmin } from '@documenso/lib/next-auth/guards/is-admin';
+import { updateUser } from '@documenso/lib/server-only/admin/update-user';
+
+import { authenticatedProcedure, router } from '../trpc';
+import { ZUpdateProfileMutationByAdminSchema } from './schema';
+
+export const adminRouter = router({
+  updateUser: authenticatedProcedure
```

Comment:
We can keep this for now but should consider add an `adminProcedure` in the future to capture these issues.

Reply:
I think now would be better than later since it looks like we're already doing this check more than once.